app/js/postEditFunction.js

```diff
@@ -7,15 +7,16 @@
  * @return Returns an object that can be used as the parameter in postUserData() function.
  */
 function createObjectForDatabase(inputClass) {
-    let formData = document.querySelectorAll(inputClass)
-    let canRetake = document.getElementById('canRetake')
-    let result = {'canRetake': '0'}
+    let formData = document.querySelectorAll(inputClass);
+    let canRetake = document.getElementById('canRetake');
```

Comment:
possibly change to this? `let canRetake = document.querySelector('#canRetake');`

prefer to use querySelector over getElementById